spec/student_spec.rb

```diff
@@ -0,0 +1,21 @@
+require_relative './spec_helper'
+
+describe Student do
+  student = Student.new(22, 'C1', 'Lucca')
```

Comment:
- [ ] Your Student spec test is failing.

  ![image](https://user-images.githubusercontent.com/54848363/212191297-6ed9788d-e3b0-425a-a903-f0dc8d5ca8bb.png)

  Here is how to fix it. You are not assigning the `name` property well in the constructor. In the Student class, change your initialize method from this:

  ```
  def initialize(age, classroom, name: 'Unknown', parent_permission: true)
    super(age, name: name, parent_permission: parent_permission)
    @id = Random.rand(1..1000)
    @classroom = classroom
  end
  ```
  to this:
  ```
  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
    super(age, name, parent_permission: parent_permission)
    @classroom = classroom
  end
  ```